vueapp/src/components/Player.vue

```diff
@@ -3,6 +3,8 @@
   <template v-slot:prepend>
     <v-icon
       :icon="player.isGuest ? 'mdi-account-box-outline' : 'mdi-account-circle'"
+      :color="player?.level?.color"
+      v-tooltip="(player.isGuest ? 'Member (' : 'Guest (') + (player?.level?.name || 'No Level') + ')'"
```

Comment:
String concatenation are bloody hard to read and hell to translate.  Use string formatting and whole phrases for each case.

Reply:
And the logic is reversed
```suggestion
      v-tooltip="(!player.isGuest ? 'Member (' : 'Guest (') + (player?.level?.name || 'No Level') + ')'"
```

Reply:
Fixed in 7890747f7080ac53682028131bcbd5513be0b3ec